edx_proctoring/models.py

```diff
@@ -173,11 +177,14 @@ class ProctoredExamStudentAttemptStatus(object):
 
     # status alias for sending email
     status_alias_mapping = {
-        submitted: _('pending'),
-        verified: _('satisfactory'),
-        rejected: _('unsatisfactory')
+        submitted: ugettext_noop('pending'),
+        verified: ugettext_noop('satisfactory'),
+        rejected: ugettext_noop('unsatisfactory')
     }
 
+    def __unicode__(self):
```

Comment:
Same issue here. I was asked to re-review in stand-up, but looks like that may have been premature.  Can you ping me when it is ready?  Thanks.